doc/tutorial/using-packages.rst

```diff
@@ -0,0 +1,93 @@
+.. _cve-2022-26809: https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-26809
+
+.. _using-packages:
+
+.. _zeek package browser: https://packages.zeek.org/
+
+################
+ Using Packages
+################
+
+Zeek contains a decent amount of functionality “out of the box” - but
+its strength comes from the package ecosystem. These packages are
+managed by the Zeek Package Manager, or ``zkg``. Then, you can find
+packages on the `Zeek Package Browser`_. This section will explain how
+to use ``zkg`` in order to get the packages you need.
+
+*********************************
+ Finding and Installing Packages
+*********************************
+
+Say you’re particularly worried about CVE-2022-26809_ (there is nothing
+special about this CVE, it’s simply an exploit that we can use for
+demonstration). Zeek does not offer any detection for this
+out-of-the-box. But, if you look on the `Zeek Package Browser`_, you can
+easily find a `package by Corelight
+<https://github.com/corelight/cve-2022-26809>`_. Let’s install it.
+
+First, since this is on the official Zeek package browser, you can
+install it with just the package name:
+
+.. code:: console
+
+   root@zeek-tutorial:/opt/zeek-training $ zkg install cve-2022-26809
+
+You may also use the git URL of the package:
+
+.. code:: console
+
+   root@zeek-tutorial:/opt/zeek-training $ zkg install https://github.com/corelight/cve-2022-26809
+
+The direct URL is necessary if the package is not in Zeek's `packages
+<https://github.com/zeek/packages>`_ repository.
+
+This will ask you to run the tests. The tests may fail, but you can
+still accept the prompt to install.
+
+Part of the setup script also cloned this repository. Try running Zeek
+on the pcap from the package:
+
+.. code:: console
+
+   root@zeek-tutorial:/opt/zeek-training $ zeek -r cve-2022-26809/testing/Traces/cve-2022-26809-4.pcap
+
+This particular package creates notices when it’s detected, but there
+isn’t a ``notice.log`` ! That's because packages have to be enabled in
+order to run. This is done differently when executing Zeek on the
+command line versus in the cluster. We’ll get to the cluster version
+later. For now, you can load the package by just including its name in
+the command line, or by prefixing that name with ``packages``:
```

Comment:
```suggestion
This particular package should create notices when traffic for the CVE is detected, but we seem to be missing a ``notice.log`` after running that command. That's because packages have to be enabled in
order to run. This is done differently when executing Zeek on the
command line versus in the cluster. We’ll get to the cluster version
later. For now, you can load the package by just including its name in
the command line, or by prefixing that name with ``packages``:
```